README.md

```diff
@@ -201,24 +201,28 @@ pip install 'polars[numpy,pandas,pyarrow]'
 | Tag        | Description                                                                  |
 | ---------- | ---------------------------------------------------------------------------- |
 | **all**    | Install all optional dependencies (all of the following)                     |
-| pandas     | Install with pandas for converting data to and from pandas DataFrames/Series |
+| adbc       | Support for reading Arrow data from SQL databases                            |
+| connectorx | Support for reading Arrow data from SQL databases                            |
+| deltalake  | Support for reading and writing Delta Lake Tables                            |
+| fastexcel  | Support for reading data from Excel files with the Calamine engine           |
+| fsspec     | Support for reading data from remote file systems                            |
 | numpy      | Install with NumPy for converting data to and from NumPy arrays              |
+| openpyxl   | Support for reading data from Excel files                                    |
+| pandas     | Install with pandas for converting data to and from pandas DataFrames/Series |
+| plot       | Support for plotting functions on DataFrames                                 |
 | pyarrow    | Reading data formats using PyArrow                                           |
-| fsspec     | Support for reading from remote file systems                                 |
-| connectorx | Support for reading from SQL databases                                       |
-| xlsx2csv   | Support for reading from Excel files                                         |
-| openpyxl   | Support for reading from Excel files with native types                       |
-| deltalake  | Support for reading and writing Delta Lake Tables                            |
-| pyiceberg  | Support for reading from Apache Iceberg tables                               |
-| plot       | Support for plot functions on DataFrames                                     |
+| pyiceberg  | Support for reading data from Apache Iceberg tables                          |
+| sqlalchemy | Support for reading Python data from SQL databases                           |
 | timezone   | Timezone support, only needed if you are on Python<3.9 or Windows            |
+| xlsx2csv   | Support for reading data from Excel files                                    |
 
-Releases happen quite often (weekly / every few days) at the moment, so updating Polars regularly to get the latest bugfixes / features might not be a bad idea.
+Releases happen quite often (weekly / every few days) at the moment, so updating Polars regularly
+to get the latest bugfixes / features might not be a bad idea.
```

Comment:
Let's not introduce line breaks like this.

Reply:
It's not a linebreak in the output; just makes it more readable in the editor. We do the same elsewhere in the file to prevent overly long lines.

Reply:
It's really not a good idea. You are making a decision for the reader that you do not need to make. Maybe I like reading my text 120 chars per line, maybe it's 60. You don't know, and you don't need to care. Just write a full sentence per line, and the user can resize their window to fit their needs.

Keeping it to one sentence per line is a concession to keep things manageable for people who like to display their readme's in a maximized window on their ultrawidescreen.